jerry-core/jerry.c

```diff
@@ -167,6 +167,18 @@ jerry_init (jerry_init_flag_t flags) /**< combination of Jerry flags */
 } /* jerry_init */
 
 /**
+ * Jerry engine initialization with custom user context
+ */
+void
+jerry_init_with_user_context (jerry_init_flag_t flags, jerry_user_context_init_cb init_cb,
+                              jerry_user_context_deinit_cb deinit_cb)
```

Comment:
Full stop after each sentence. Please add a comment for each function. Coding style is in #1459.